frontend/cypress/integration/common/overview.ts

```diff
@@ -125,6 +125,26 @@ When(`user selects {string} time range`, (interval) => {
         .should('not.exist');
 });
 
+When(`user selects {string} traffic duration`, (duration) => {
```

Comment:
Typo, I guess it should be direction.

I guess we can add the type to the parameter? (direction: string)